README.md

```diff
@@ -2,47 +2,64 @@
-```
+The Codacy Coverage Reporter GitHub Action:
 
-> Get the latest version from the [Releases page](https://github.com/codacy/codacy-coverage-reporter-action/releases).
+-   Uploads coverage report data to Codacy.
+-   Prints the coverage report upload status on the console, which is visible on the workflow panel of the GitHub Action.
+-   Fails the workflow if code coverage falls below your project’s [quality settings](https://docs.codacy.com/repositories-configure/adjusting-quality-settings/#gates).
 
-## Workflow options
+To use the GitHub Action with default settings:
 
-Change these options in the workflow `.yml` file to meet your GitHub project needs:
+1.  Set up an API token to allow the GitHub Action to authenticate on Codacy:
 
-| Setting                           | Description                                                                            | Recommended value                     |
-| --------------------------------- | -------------------------------------------------------------------------------------- | ------------------------------------- |
-| `api-token`                       | [Account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) | `${{ secrets.CODACY_API_TOKEN }}`     |
-| `project-token`                   | [Project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) | `${{ secrets.CODACY_PROJECT_TOKEN }}` |
-| `coverage-reports`                | Optional comma-separated list of reports to send                                       | `''`                                  |
-| `language`                        | Optionally force associating a language with your coverage report                      | `''`                                  |
-| `force-coverage-parser`           | Optionally force using a specific coverage report parser                               | `''`                                  |
+    -   **If you're setting up one repository**, [obtain a project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
+    -   **If you're setting up multiple repositories**, [obtain an account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
+
+    > ⚠️ **Never write API tokens to your configuration files** and keep your API tokens well protected, as they grant owner permissions to your projects on Codacy.
+
+2.  [Generate a supported code coverage report](https://docs.codacy.com/coverage-reporter/#adding-coverage-to-your-repository) before each push to your repository.
```

Comment:
* I think it's more appropriate to link directly to the section on how to generate the coverage reports (just like in the first sentence of this section).
* The tests that generate the coverage reports should run in the CI/CD pipeline that's triggered by each push to the repository. So, it's more correct to say that the user must generate the coverage reports "on each push" instead of "before each push" to the repository.

```suggestion
2.  [Generate a supported code coverage report](https://docs.codacy.com/coverage-reporter/#generating-coverage) on each push to your repository.
```